README.md

```diff
@@ -0,0 +1,328 @@
+# Play-Mailer
+
+[![Build Status](https://travis-ci.org/playframework/play-mailer.svg?branch=master)](https://travis-ci.org/playframework/play-mailer)
+
+Play Mailer is a powerful Scala Mailing library. It provides a simple configurable mailer.
+
+> For Play 2.4.x or lower, please read the README on [the older 5.x branch.](https://github.com/playframework/play-mailer/blob/4bdc0cd58e0978f289891807d3ac158ce620e81f/README.adoc)
+ 
+## Getting Started
+
+To get started you add `play-mailer` and `play-mailer-guice` as a dependency in SBT:
+
+```scala
+libraryDependencies += "com.typesafe.play" %% "play-mailer" % "6.0.0"
+libraryDependencies += "com.typesafe.play" %% "play-mailer-guice" % "6.0.0"
+```
+
+After that you need to configure the mailer inside your `application.conf`:
+ 
+```HOCON
+play.mailer {
+  host = "example.com" // (mandatory)
+  port = 25 // (defaults to 25)
+  ssl = no // (defaults to no)
+  tls = no // (defaults to no)
+  tlsRequired = no // (defaults to no)
+  user = null // (optional)
+  password = null // (optional)
+  debug = no // (defaults to no, to take effect you also need to set the log level to "DEBUG" for the application logger)
+  timeout = null // (defaults to 60s in milliseconds)
+  connectiontimeout = null // (defaults to 60s in milliseconds)
+  mock = no // (defaults to no, will only log all the email properties instead of sending an email)
+}
+```
+
+If you are on Play 2.6.x you can skip to [Usage](#Usage).
+For Play 2.5.x you might also need a ConfigModule:
+
+### Scala 2.5.x
```

Comment:
Oh, got it. This is the Scala example.

Anyway, I think this title could be clearer: "Play 2.5.x and Scala".